lib/rules/indentation/__tests__/rules.js

```diff
@@ -10,7 +10,7 @@ testRule(rule, {
 
 	accept: [
 		{
-			code: '/* anything\n' + '    goes\n' + '\t\t\twithin a comment */\n' + '',
+			code: String('/* anything\n' + '    goes\n' + '\t\t\twithin a comment */\n'),
```

Comment:
This and the next change this in this, I'm not sure it was the original intention since it's already a string?